showcase/config/webpack-custom-config.dev.json

```diff
@@ -1,4 +1,4 @@
 {
-  "cspConnectSrc": "http://localhost:5000 http://localhost:5001 http://localhost:5002 http://localhost:4000 https://nationalbankbelgium.github.io",
+  "cspConnectSrc": "http://localhost:5000 http://localhost:5001 http://localhost:5002 http://localhost:4000 https://nationalbankbelgium.github.io ws://localhost:4200",
```

Comment:
In fact, that was my bad... I think we should not modify the `HOST` and `PORT` via environment variables anymore because NG CLI already allows this via an option

Reply:
If we can get the default port and default host or the values defined in **angular.json** (if defined), I think it would be better to keep the previous scenario in `cspDirectives` in webpack-partial.dev.js --> `"connect-src 'self' ws://" + METADATA.HOST + ":" + METADATA.PORT + " " " + webpackCustomConfig["cspConnectSrc"]`

Reply:
True, I'm fixing this